services/ResourceService/src/index.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Islandora\ResourceService;
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+use Silex\Application;
+use Islandora\Chullo\Chullo;
+use Islandora\Chullo\TriplestoreClient;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Psr\Http\Message\ResponseInterface;
+use Silex\Provider\TwigServiceProvider;
+use Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory;
+
+
+date_default_timezone_set('UTC');
+
+$app = new \Silex\Application();
+
+$app['debug'] = true;
+
+$app->register(new \Silex\Provider\TwigServiceProvider(), array(
+  'twig.path' => __DIR__.'/../templates',
+));
+
+$app['twig'] = $app->share($app->extend('twig', function($twig, $app) {
+
+  return $twig;
+}));
+
+$app['fedora'] =  $app->share(function() {
```

Comment:
No need to call share() here.  Just passing in the closure is enough.  In earlier versions you had to do because default was to create a new service every time it was pulled from the DI container.  Now the default is to share.